src/main.rs

```diff
@@ -21,18 +24,23 @@ fn match_input (guess: u32, secret: u32) -> bool {
         Ordering::Greater => println!("lower!"),
         Ordering::Equal   => println!("bingo! 🎉"),
     }
-    return guess.eq(&secret);
+
+    guess.eq(&secret)
 }
 
 fn ask_number () -> u32 {
-    println!("Please input your guess : ");
+    loop {
```

Comment:
Created a loop, where in the question will be repeated until the user provides a guess.